validation-test/Reflection/reflect_UInt16.swift

```diff
@@ -4,7 +4,7 @@
 
 // RUN: %target-run %target-swift-reflection-test %t/reflect_UInt16 | %FileCheck %s --check-prefix=CHECK-%target-ptrsize
 
-// REQUIRES: objc_interop
+// REQUIRES: objc_interop || (asserts && OS=linux-gnu)
```

Comment:
A couple points on this:
1) We're not going to want to copy "objc_interop || (asserts && OS=linux-gnu)" into other test files. If we do, we'll have to change many files when adding reflection support for another OS.
2) 'swift_stdlib_asserts' seems closer to what we want than 'asserts'?

I suggest adding a feature in test/lit.cfg called 'basic_reflection_support', like: `if hasFeature('objc_interop') or (hasFeature('swift_stdlib_asserts') and not kIsWindows): ...`. Then you can simply use 'REQUIRES: basic_reflection_support' here. Btw, it appears that StdlibUnittest.swift imports 'Glibc' when compiled for *BSD, so hopefully it's only Windows that requires some extra work.

Reply:
@vedantk If we're just checking if the OS is not Windows, should we update the #if in SwiftReflectionTest to match this as well?

Reply:
Yes, please see if you can replace '#elseif os(Linux)' with something that's consistent with the way the 'basic_reflection_support' lit feature is defined, perhaps '#elseif !os(Windows)'.